robocop/checkers/comments.py

```diff
@@ -7,21 +7,31 @@
 from robot.utils import FileReader
 
 from robocop.checkers import RawFileChecker, VisitorChecker
-from robocop.rules import Rule, RuleSeverity
+from robocop.rules import Rule, RuleSeverity, RuleParam
 from robocop.utils import ROBOT_VERSION
 
 rules = {
     "0701": Rule(
+        RuleParam(
+            name="markers",
+            default="todo,fixme",
+            converter=str,
+            desc="List of case insensitive markers that violate the rule in comments.",
+        ),
         rule_id="0701",
         name="todo-in-comment",
-        msg="Found {{ todo_or_fixme }} in comment",
+        msg="Found a marker '{{ marker }}' in the comments",
         severity=RuleSeverity.WARNING,
         docs="""
         Example::
         
             # TODO: Refactor this code
-            # fixme 
+            # fixme
         
+        Configuration example::
+
+            robocop --configure "todo-in-comment:markers:todo,fixme,Remove me,Fix this!"
```

Comment:
I think the quotes are not needed here for rule configuration. Can you please verify and remove if redundant?

Reply:
Using a phrase with spaces on it needs quotes for the configuration.
What do you think? Should we add two config examples with and without phrases? 

`robocop --configure todo-in-comment:markers:todo,fixme,TBD`
and
`robocop --configure "todo-in-comment:markers:Remove me,Fix this!"`

Reply:
Oh, sorry. I missed there are spaces inside. Good catch!
@bhirsz I think we should add this info to our documentation.
@rikerfi I think you can keep it like it is, no need to update it. Leave it with quotes and spaces.

Reply:
@mnojek Sure, I will also separably put this information to our docs in some common page (how to provide values with spaces).